examples/01-Getting-started.ipynb

```diff
@@ -71,9 +71,11 @@
      "name": "stderr",
```

Comment:
<p>having regression metrics printed out for classification task would be confusing for the user. I proposed a solution above to avoid that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>